### PR TITLE
Style mobile menu

### PR DESCRIPTION
**Pull request by kiboma2021**
Style mobile menu with DOM Javascript

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,430 +4,336 @@
...
+            <a href="#about" class="nav_link">About </a>
+          </li>
+          <li class="nav_item">
+            <a href="#contact" class="nav_link">Contact</a>
+          </li>
+        </ul>
+        <div class="hamburger">
+          <span class="bar"> </span>
+          <span class="bar"> </span>
+          <span class="bar"> </span>
```
Comment (gtekle) [inline index.html:29]: - [ ] Kindly remove the white gap at the top when mobile menu is open.
- [ ] Kindly add some space at the top for the menu close button as in the Figma design
#### Design
mobile-menu-top-figma
#### Impl
mobile-menu-top-white-strip

File: index.html [hunk 2]
```diff
@@ -4,430 +4,336 @@
...
-              Glad to see you!</h1>
+        <div class="row">
+          <div class="intro1 col-12">
+            <h1>
+              I’m Omayio <br />
+              Glad to see you!
+            </h1>
           </div>
-          <div class="flexbox-item flexbox-item-2">           
+          <div class="intro2 col-12">
```
Comment (gtekle) [inline index.html:43]: - [ Optional ] The headline section is skewed on mobile version. I believe it is due to the desktop image used for mobile. Kindly rectify for better look and feel of your portfolio.
#### Impl
mobile-headline-section
#### Design
mobile-headline-section-figma

File: index.html [hunk 3]
```diff
@@ -4,430 +4,336 @@
...
-        <div class="grid-item show-case2">
-          <div class="content">
-            <h4>Multi-Post Stories</h4>
-            <ul class="flex">
-              <li class="cano">CANOPY</li>
-              <li>Back End Dev</li>
-              <li>2015</li>
-            </ul>
-          </div>
+        <div class="contact-item-2">
```
Comment (gtekle) [inline index.html:290]: - [ Optional ] The contact section is skewed in mobile version and I believe it is due to the background image for desktop used here for mobile as well. Can we rectify this?
#### Design
mobile-contact-page-figma
#### Imp
mobile-contact-page
l

File: index.html [hunk 4]
```diff
@@ -4,430 +4,336 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Ben Omayio</title>
+    <title>Ben Omayio Portfolio</title>
     <link rel="stylesheet" href="styles.css" />
   </head>
+
   <body>
```
Comment (gtekle) [inline index.html:11]: - [ Optional ] The is scroll effect while the mobile menu is open. what do you think stopping scroll effect better look and feel? Take a look at this snapshot taken after a bit of down scroll.
mobile-body-scroll-on-open-menu

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @kiboma2021 :wave: , 

## You have done a great job implementing the mobile menu! :clap: :clap: :clap: 

There are some issues that you still need to work on to go to the next project but you are almost there!
this-close


## Highlights
:heavy_check_mark: Professional Readme :100: 
:heavy_check_mark: Meaningful commit messages
:heavy_check_mark: Good use of Github flow

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please check the inline comments below

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi Team,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request closed without merging**